### PR TITLE
PHPFOL-8 docs: add code-of-conduct.md and contributing.md

### PR DESCRIPTION
**Pull request by toutvukantabu**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+Public or private harassment
+Publishing others’ private information, such as a physical or email address, without their explicit permission
+Other conduct which could reasonably be considered inappropriate in a professional setting
+Enforcement Responsibilities
+Community leaders are responsible for clarifying and enforcing our standards of acceptable behavior and will take appropriate and fair corrective action in response to any behavior that they deem inappropriate, threatening, offensive, or harmful.
+
+Community leaders have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct, and will communicate reasons for moderation decisions when appropriate.ing
+
+
+## Conventions
```
Comment (JasonBenett) [inline CONTRIBUTING.md:49]: Conventions are not about how to use Git. It's about how to write the code. We can leave it empty if you have no idea currently and fill it each time we define a rule.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
+# Contributing to PHPFOLIO
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+
+#### Table Of Contents
+
+[Code of Conduct](#code-of-conduct)
+
+[standards](#standards)
```
Comment (JasonBenett) [inline CONTRIBUTING.md:10]: Missing uppercase on first letter.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+
+
+
+## Code of Conduct
+We take our open source community seriously and hold ourselves and other contributors to high standards of communication. By participating and contributing to this project, you agree to abide by our communication standards. 
+Here are the details : [Code of Conduct](https://github.com/JasonBenett/phpfolio/blob/main/CODE-OF-CONDUCT.md).
+
+
+## STANDARDS
```
Comment (JasonBenett) [inline CONTRIBUTING.md:27]: No full uppercase please. Consistency between the titles.

**Review by JasonBenett — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 3 above)
Comment (Coralie-BDY) [reply CONTRIBUTING.md:27]: Shouldn't he put the 1st hook next to the start of what it frames?

File: CONTRIBUTING.md (hunk 1 above)
Comment (toutvukantabu) [reply CONTRIBUTING.md:49]: yes that's what I thought, we improve it as we go, the main thing is to have a low

**Review by JasonBenett — APPROVED** [review]

**Pull request merged**